internal/uvm/create_lcow.go

```diff
@@ -33,14 +42,18 @@ const (
 type OptionsLCOW struct {
 	*Options
 
-	BootFilesPath         string  // Folder in which kernel and root file system reside. Defaults to \Program Files\Linux Containers
-	KernelFile            string  // Filename under BootFilesPath for the kernel. Defaults to `kernel`
-	KernelDirect          bool    // Skip UEFI and boot directly to `kernel`
-	RootFSFile            string  // Filename under BootFilesPath for the UVMs root file system. Defaults are `initrd.img` or `rootfs.vhd` based on `PreferredRootFSType`.
-	KernelBootOptions     string  // Additional boot options for the kernel
-	EnableGraphicsConsole bool    // If true, enable a graphics console for the utility VM
-	ConsolePipe           string  // The named pipe path to use for the serial console.  eg \\.\pipe\vmpipe
-	SCSIControllerCount   *uint32 // The number of SCSI controllers. Defaults to 1 if omitted. Currently we only support 0 or 1.
+	BootFilesPath         string             // Folder in which kernel and root file system reside. Defaults to \Program Files\Linux Containers
+	KernelFile            string             // Filename under BootFilesPath for the kernel. Defaults to `kernel`
+	KernelDirect          bool               // Skip UEFI and boot directly to `kernel`
+	RootFSFile            string             // Filename under BootFilesPath for the UVMs root file system. Defaults are `initrd.img` or `rootfs.vhd` based on `PreferredRootFSType`.
+	KernelBootOptions     string             // Additional boot options for the kernel
+	EnableGraphicsConsole bool               // If true, enable a graphics console for the utility VM
+	ConsolePipe           string             // The named pipe path to use for the serial console.  eg \\.\pipe\vmpipe
+	SCSIControllerCount   *uint32            // The number of SCSI controllers. Defaults to 1 if omitted. Currently we only support 0 or 1.
+	ExecCommandLine       string             // The command line to exec from init. Defaults to GCS
+	ForwardStdout         *bool              // Whether stdout will be forwarded from the executed program
```

Comment:
Can you specify the defaults in the comments here. Makes it a lot easier to understand